src/hazmat/sieve.rs

```diff
@@ -41,7 +41,9 @@ where
     let bit_length = bit_length.get();
 
     let mut random = T::try_random_bits(rng, bit_length).map_err(|err| match err {
-        RandomBitsError::RandCore(_) => unreachable!("`rng` impls `CryptoRng` and therefore is infallible"),
+        RandomBitsError::RandCore(_) => {
+            unreachable!("`rng` impls `CryptoRng` and therefore is infallible")
+        }
```

Comment:
There are a few of these white space changes that seem irrelevant to the rest of the PR?

Reply:
Yeah I messed up the formatting bad on that one. I'll adjust that.